### PR TITLE
Queries

### LINKED ISSUE #11: database / query functions
make getData and postData functions

### PR DESCRIPTION
**Pull request by imansedky5**
make getData, postData and  updateData files

### CODE REVIEW

File: src/database/queries/postData.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const postComments = ({ commentContent }) => {
+    const sql = {
+        text: 'INSERT INTO comments(comment_content) VALUES($1)',
+        values: [commentContent],
+    };
+    return connection.query(sql);
+};
+
+module.exports(postPosts, postComments);
```
Comment (AlaaShurrab) [inline src/database/queries/postData.js]: change the brackets from () to {}

Comment (mAbadsa) [inline src/database/queries/postData.js]: module.exports = { postPosts, postComments };

File: src/database/queries/updataData.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const connection = require('../config/connection');
+
+const updatePost = ({ postId }, { postContent }) => connection.query(`UPDATE posts SET post_content = ${postContent} WHERE id = ${postId}`);
```
Comment (mAbadsa) [inline src/database/queries/updataData.js]: use $[-any number] instead of ${var}
updatePost parameters (postId, postContent);

**Review by AlaaShurrab — APPROVED** [review]

File: src/database/queries/updataData.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const connection = require('../config/connection');
+
+const updatePost = (postId, postContent) => {
+    const sql = {
+        text: 'UPDATE posts SET post_content = $2 WHERE id = $1',
+        values: [postId, postContent],
+    };
+    return connection.query(`UPDATE posts SET post_content = ${postContent} WHERE id = ${postId}`);
```
Comment (mAbadsa) [inline src/database/queries/updataData.js]: put sql instead of `UPDATE posts SET post_content = ${postContent} WHERE id = ${postId}`

**Review by mAbadsa — APPROVED** [review]

**Pull request merged**